max_covariance: 0.2
finish: 5

node0:
    x: 8.2
    y: 8.15

node1:
    x: 10
    y: 8.1

node2:
    x: 10.5
    y: 8.1

node3:
    x: 11.3
    y: 8.6

node4:
    x: 11.35
    y: 9.3

node5:
    x: 11.2
    y: 11.9

link0_1:
    enabled: 1
    orient: 90
    orient_y: 8.6
    orient_k: -1
    orient_a: -0.01
    orient_d: 0.6
#    marker_a: 45

link1_2:
    enabled: 1

link2_3:
    enabled: 1

link3_4:
    enabled: 1

link4_5:
    enabled: 1
    orient: 90
    orient_x: 10.5
    orient_k: 1
    orient_a: 1.57
    orient_d: 0.8
    marker_a: 25
    marker_d: 1
